Bisulfite_QC_CpGretentiondistribution.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+## make sure the following is in PATH
+## biscuit samtools, bedtools, awk
+
+function biscuitQC
+
+{
+  ## simple test to make sure all is available
+  if [[ `which biscuit 2>&1 >/dev/null` ]]; then echo "biscuit does not exist in PATH"; exit 1; fi
+  if [[ `which samtools 2>&1 >/dev/null` ]]; then echo "samtools does not exist in PATH"; exit 1; fi
+  if [[ `which bedtools 2>&1 >/dev/null` ]]; then echo "bedtools does not exist in PATH"; exit 1; fi
+  if [[ `which awk 2>&1 >/dev/null` ]]; then echo "awk does not exist in PATH"; exit 1; fi
+  for var in BISCUIT_CPGBED BISCUIT_CGIBED BISCUIT_RMSK BISCUIT_EXON BISCUIT_GENE BISCUIT_TOPGC_BED BISCUIT_BOTGC_BED input_bam input_vcf; do
+    if [[ ${!var} != "<unset>" && ! -f ${!var} ]]; then
+      >&2 echo "$var: ${!var} does not exist."
+      exit 1;
+    fi
+  done
+
+  echo "Running."
+  set -xe
```

Comment:
Probably not a bad idea to make this  `set -xeou pipefail` instead.  That way, if one of the steps that gets piped to another step fails, the whole thing will die.  As is, there are a couple of places where biscuit or bedtools could fail and downstream steps could succeed, and the workflow would succeed, but with weird or truncated results (this applies to all the QC bash scripts)